### ISSUE TITLE
Sync Beta 30 breaks link handling

### ISSUE THREAD

**Issue by Coffee-Viking** [zacharee/MastodonRedirect#15]
Sync implemented link handling for the top 50 instances internally with beta 30 (2023-08-15), and ever since Lemmy Redirect will instead open to the dashboard (default app selector) instead of actually pushing the link to Sync, even if Sync has not been enabled for link handling.

**Comment by Coffee-Viking** [comment]
Just updated to 1.3.7, issue persists.

**Comment by zacharee** [comment]
I'll take a look once the beta is public.

**Comment by Coffee-Viking** [comment]
@zacharee I got it from google play, shouldn't be long until it's rolled out everywhere. Was released a few hours ago.

**Comment by zacharee** [comment]
If you're an internal tester, you get it instantly. I have to wait until Google approves the beta.

If you send me the APK, I can try updating the launch action and uploading a test release.

**Comment by Coffee-Viking** [comment]
Got it, I'll email you :-)

**Comment by zacharee** [comment]
Try this: https://github.com/zacharee/MastodonRedirect/releases/tag/1.3.7-sync-test

**Comment by Coffee-Viking** [comment]
Seems to persist for me... The sync link handler opens in the background
but freezes.

Screen recording: https://imgur.com/a/17G3eP1

Might also be an issue on Sync's end though, I've also opened an issue on
the bug tracker there.

**Comment by zacharee** [comment]
I saw mentions in the Discord of Sync's link handler freezing and looping.

**Comment by Coffee-Viking** [comment]
Yep I'm on the discord as well, just figured I'd open all relevant issues
so that nothing gets lost.
